pom.xml

```diff
@@ -90,7 +90,7 @@
     <log4j.version>1.2.17</log4j.version>
 
     <!-- Spark version -->
```

Comment:
minor note: I changed this line to `Flink version` in #10, so not changing it in this PR to avoid conflicts.